server/statistics/hot_peer_cache_test.go

```diff
@@ -435,28 +450,38 @@ func (t *testHotPeerCache) TestRemoveFromCacheRandom(c *C) {
 				region := buildRegion(Write, peerCount, interval)
 
 				target := uint64(10)
-				movePeer := func() {
+				intervalSums := make(map[uint64]int)
+				step := func(i int) {
 					tmp := uint64(0)
-					tmp, region = schedule(c, removeReplica, region)
-					_, region = schedule(c, addReplica, region, target)
-					target = tmp
+					if i%5 == 0 {
+						tmp, region = schedule(c, removeReplica, region)
+					}
+					rets := checker(c, cache, region)
+					checkIntervalSumContinuous(c, intervalSums, rets, interval)
+					if i%5 == 0 {
+						_, region = schedule(c, addReplica, region, target)
+						target = tmp
+					}
 				}
+
 				// prepare with random move peer to make the interval sum of peers are different
```

Comment:
Does the comment need to be changed?

Reply:
I don't think we need to change it because the actual move peer is to add replica in the target store and then remove replica from the source store